KinConnections/templates/connector.html

```diff
@@ -101,7 +123,7 @@ <h3><span class="name">{{connector['first_name']}} {{connector['last_name']}}</s
             <h4 style="font-weight: 600;">Connect with {{connector['first_name']}} {{connector['last_name']}}</h4>
             <p>To connect, use the following text box to send the connector a message! Please ensure you are following the <a>KinConnections Connecting Guidelines</a>. You can also select a message template by clicking the buttons below.</p>
 
-            <form action="loadingBlocker()" method="post" role="form" class="php-email-form">
+            <form action="/send_email/{{connector.get('id', None)}}" method="POST" role="form" id="connections-form" class="php-email-form">
```

Comment:
Why do we have both the `AJAX` call above with the event listener and also the `action` attribute on the form? We can probably get rid of one or the other. Help clean up the redundant code a little.

Reply:
Both are present because they work asynchronously to send the email message. The loadingBlocker() function shows a buffering circle until the message is sent whereas the AJAX function actually sends the email and tells the loadingBlocker() to show success or error.